src/components/themeSwitch/index.scss

```diff
@@ -1,7 +1,7 @@
 @import "src/style/mixins/index";
 
 .theme-switch {
-  padding: 1rem;
+  padding: 0.5rem;
```

Comment:
Non blocking question: Was this done to leave a maximum of free space on mobile?

Reply:
Yep, and to respect the focus size without both the logo and switch being to close to each other.